tests/e2e/authoring-form.spec.js

```
@@ -0,0 +1,299 @@
+  test('a tag that does not exist yet can be made from the field', async ({ page }) => {
+    await stubBackend(page)
+    await openForm(page)
+
+    // Drupal's tags widget creates a term for anything typed that does not
+    // match, which is why tagging feels like typing rather than picking.
+    const tags = page.getByTestId('reference-input').first()
+    await tags.fill('Sourdough')
+    await expect(page.getByTestId('reference-create')).toContainText('Sourdough')
+    await page.getByTestId('reference-create').click()
+
+    const staged = await page.evaluate(() => window.$nuxt.$store.getters['authoringCart/resources'])
```

ESLint

[error] 'window' is not defined.

(no-undef)